spec/admin.mediawiki

```diff
@@ -16,13 +16,15 @@ possible.
 {|
 ! variable !! relevant section !! units || default
 |-
-| epoch&nbsp;duration || [[fundamentals.mediawiki/#Epochbased_Order_Matching|Epoch-based Order Matching]] || milliseconds || 60000
+| epoch&nbsp;duration || [[fundamentals.mediawiki/#Epochbased_Order_Matching|Epoch-based Order Matching]] || milliseconds || 60,000
```

Comment:
I just added a comma here, but I did notice that we don't have a default value, and it appears to me that we should be checking for a zero value in `loadMarketConf`.

Reply:
Fair enough, will check for 0 there.  Also seems reasonable to replace 0/-1/omitted with a the specd default.